lib/ansible/plugins/action/dnos10_template.py

```diff
@@ -1,25 +0,0 @@
-# This file is part of Ansible
```

Comment:
The _template files appear to have been `git mv` in a strange way, I don't see a `dellos6_template.py` file

e.g
dnos10_config.py  -> dellos9_config.py